specification/batch/data-plane/Microsoft.Batch/2017-06-01.5.1/BatchService.json

```diff
@@ -12412,13 +12412,14 @@
         },
         "validationStatus": {
           "type": "string",
-          "title": "Whether the task counts have been validated. If the validationStatus is unvalidated, then the Batch service has not been able to check state counts against the task states as reported in the List Tasks API. The validationStatus may be unvalidated if the job contains more than 200,000 tasks.",
+          "title": "Whether the task counts have been validated. If the validationStatus is unvalidated, then the Batch service has not been able to check state counts against the task states as reported in the List Tasks API.",
+          "description": "The validationStatus may be unvalidated if the job contains more than 200,000 tasks.",
```

Comment:
I updated the document (moved some text around) to help clarify.

`Validated` == Counts have been validated and confirmed to match the result of the List Tasks API.
`Unvalidated` == Counts have not been validated, and may not exactly match the result of the List Tasks API.